directory.py

```diff
@@ -42,11 +44,37 @@ def user_from_identity(sub) -> Dict[str, Any]:
         raise UserNotFoundError
 
 
+def insert_todo(todo: Todo):
+    try:
+        ds().set_object(
+            object_type="resource", object_id=todo.ID, display_name=todo.Title, properties={}
+        )
+
+        ds().set_relation(
+            subject_id=todo.OwnerID,
+            subject_type="user",
+            object_id=todo.ID,
+            object_type="resource",
+            relation="owner",
+        )
+    except NotFoundError:
```

Comment:
What condition is this except block attempting to handle? When would set_object or set_relation return NotFoundError?